yarn.lock

```diff
@@ -7791,7 +7788,7 @@ file-loader@^3.0.1:
     loader-utils "^1.0.2"
     schema-utils "^1.0.0"
 
-file-saver@eligrey/FileSaver.js#1.3.8:
+"file-saver@github:eligrey/FileSaver.js#1.3.8":
```

Comment:
I search half an hour for the root of this strange diff but I did not find anything. Could you please try a `yarn cache clean && yarn install`? Maybe we have luck with that